draft-ietf-secevent-subject-identifiers.md

```diff
@@ -41,6 +41,12 @@ normative:
     target: http://www.iana.org/assignments/jwt
     author:
       org: IANA
+  DID:
+    title: Decentralized Identifiers (DIDs) v1.0
+    target: https://www.w3.org/TR/did-core/
```

Comment:
This may not be the correct durable link, because W3C might publish a later version of this spec. I suggest to consult with the W3C liaison, @wseltzer.

Reply:
I believe this is the recommended stable URL for the document, but I am happy to change it if there's a better one.

Reply:
thanks, /TR links are stable